src/app/components/server/BreachLogo.module.scss

```diff
@@ -0,0 +1,112 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@import "../../tokens";
+
+.breachLogo {
+  height: 1.75rem;
+  display: inline-block;
+}
+
+span.breachLogo {
+  line-height: 1.75rem;
+  width: 1.75rem;
+  border-radius: 1.75rem;
+  font-size: 1rem;
+  font-weight: bold;
+  text-transform: uppercase;
+  text-align: center;
+
+  &.purple40 {
```

Comment:
This file is "new" because I changed it from `.css` to `.scss`, but the `&.<colour>` classes are what's actually new, incorporating the colours that were first confined to `getRandomLightNebulaColor` in the dashboard page.